src/mqtt/qmqtt_client_p.cpp

```diff
@@ -175,6 +176,13 @@ void QMQTT::ClientPrivate::initializeErrorHash()
     _socketErrorHash.insert(QAbstractSocket::SslInternalError, SocketSslInternalError);
     _socketErrorHash.insert(QAbstractSocket::SslInvalidUserDataError, SocketSslInvalidUserDataError);
     _socketErrorHash.insert(QAbstractSocket::TemporaryError, SocketTemporaryError);
+
+    _mqttErrorHash.insert(MqttError::MqttUnacceptableProtocolVersionError, MqttUnacceptableProtocolVersionError);
+    _mqttErrorHash.insert(MqttError::MqttIdentifierRejectedError, MqttIdentifierRejectedError);
+    _mqttErrorHash.insert(MqttError::MqttServerUnavailableError, MqttServerUnavailableError);
+    _mqttErrorHash.insert(MqttError::MqttVadUserNameOrPasswordError, MqttVadUserNameOrPasswordError);
+    _mqttErrorHash.insert(MqttError::MqttNotAuthorizedError, MqttNotAuthorizedError);
+
```

Comment:
waste

Reply:
Maintain the same processing mode as QAbstractSocket.